UPGRADING.md

```diff
@@ -1,316 +1,277 @@
+| `getMetadata` / `getMetadataFromSection`                           | Retrieves previously set metadata from a section/tab. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#global-metadata) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#global-metadata) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#global-metadata)).
+| `setUser:withEmail:andName` / `setUser(_:email:name)`              | Sets the active user for the app for future events. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#adding-user-data) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#adding-user-data) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#adding-user-data)).
 
-- config.notifierType
-+ config.appType
+#### Changes
 
-- config.notifyReleaseStages
-+ config.enabledReleaseStages
+The following changes have been made to the `Bugsnag` client: 
 
-- config.setEndpoints(notify: sessions)
-+ config.setEndpoints(BugsnagEndpointConfiguration(notify: sessions))
-```
+##### Objective-C
 
-#### Removals
-
-```diff
-- BugsnagBeforeNotifyHook
-- config.metadata
-- config.config
-- config.breadcrumbs
-- config.reportBackgroundOOMs
-- config.notifyURL
-- config.sessionURL
-- config.shouldAutoCaptureSessions
-- config.autoNotify
-- config.shouldSendReports
-- config.errorApiHeaders
-- config.sessionApiHeaders
-- config.codeBundleId
-```
+| v5.x API                                                           | v6.x API                                                          |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------- |
+| `addAttribute:withValue:toTabWithName:`                            | `addMetadataToSection:withKey:toSection:`                         |
+| `clearTabWithName`                                                 | `clearMetadataFromSection`                                        |
+| `startBugsnagWithApiKey`                                           | `startWithApiKey`                                                 |
+| `startBugsnagWithConfiguration`                                    | `startWithConfiguration`                                          |
+| `stopSession`                                                      | `pauseSession`                                                    |
 
-### `Bugsnag` class
+##### Swift
 
-#### Removals
+| v5.x API                                                           | v6.x API                                                          |
```

Comment:
perhaps it would make sense to add this as an extra column onto the first table to reduce duplication?

Reply:
Merged tables using the style I've used further down.